packages/grid/_modules_/grid/hooks/features/core/gridState.ts

```diff
@@ -33,28 +33,28 @@ import {
 } from '../virtualization/renderingState';
 
 export interface GridState {
-  rows: InternalGridRowsState;
-  editRows: GridEditRowsModel;
-  pagination: GridPaginationState;
-  options: GridOptions;
-  isScrolling: boolean;
   columns: GridInternalColumns;
+  columnMenu: GridColumnMenuState;
```

Comment:
I understand why we need to keep the options in the state but I'm a but concerned that we are introducing an extra step here. My rationale is that if I am providing options to the grid I'm not really interested in the state, i just want to force it. What would be the difference between having options and just exposing the entire state (which is already) the case and making developers use that to "control" it. It seems like 2 different ways to do the same thing in the end. Maybe we can consider dropping one of the 2 options (probably hide the state).

Reply:
Just to be clear, the state has not changed, I just reordered it alphabetically.

Reply:
Then maybe what we should learn from this thread is that it would have been clearer to do the sorting in a dedicated PR.

Reply:
ok I revert